Makefile

```diff
@@ -0,0 +1,11 @@
+# It seems that the venv doesn't work in the makefile. https://stackoverflow.com/questions/44052093/makefile-with-source-get-error-no-such-file-or-directory
```

Comment:
The problem in that Stackoverflow question appears to be the use of the Shell directive `source` and has an accepted [answer](https://stackoverflow.com/questions/44052093/makefile-with-source-get-error-no-such-file-or-directory/53936226#53936226) which suggests to use the _"dot"_-notation (`.`) instead. Did you try that?

There seem to be plenty of examples out there that use virtual environments in a `Makefile`
- https://blog.byronjsmith.com/makefile-shortcuts.html
- https://stackoverflow.com/questions/24736146/how-to-use-virtualenv-in-makefile
- https://blog.horejsek.com/makefile-with-python

Reply:
thanks.